### DISCUSSION TITLE
[2470300]水烟水雾

### DISCUSSION THREAD

**Discussion by ljjjjjjjooo** [pjy612/SteamManifestCache#737] [Share]
### Steam 游戏ID

`2470300`

### 密钥列表

```hash
2470301;a7872872c156d69ba77a8005eaf9974f9a5917f2b68d33fd1aad07b93d827730
```
### 清单附件

[2470300.zip](https://github.com/user-attachments/files/16311786/2470300.zip)

### 备注

email;748918756@qq.com

**Comment by pjy612** [comment]
* 鸽 内推
